backend/src/site/main.js

```diff
@@ -95,6 +95,9 @@ exports.run = function (database, getQueueChannel) {
     app.get('/splits', sessionChecker, (req, res) => {
         res.sendFile(root + '/index.html');
     });
+    app.get('/request-confirmation', sessionChecker, (req, res) => {
```

Comment:
remove sessionChecker, move whole line to somewhere more appropriate where the other routes don't check session